contents/product-engineers/best-customer-retention-metrics.md

```diff
@@ -0,0 +1,246 @@
+
+## 5. Customer satisfaction score (CSAT)
+
+### What is a CSAT?
+
+Your customer satisfaction score, or CSAT, is similar to your NPS, but is specifically a great customer retention metric to use before you’ve found product-market fit. Satisfaction tends to be short-lived, whereas loyalty takes time to build. And that’s why CSAT is a great metric for early-stage startups.
+
+### How is it useful?
+
+Any businesses that interact in a very direct, tangible way with customers should be measuring their CSAT: think retailers, restaurants, and customer support. On the flipside, businesses that rely on very transactional relationships with customers (gas stations) or high-value, single-purchase businesses (car dealerships) should place higher importance on other metrics.
+
+### How to calculate your CSAT score
+
+Again, ask a straightforward question: ‘How simple are you with our product?’ and give people a range from 0 to 10 or 0 to 5 – or use smiley faces to represent a 0 to 5 scale. You should collect CSAT responses right after someone uses a specific feature.
+
+![CSAT survey](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/nps-csat-ces/csat-survey.png)
+
+Here’s the math. Positive responses sit in the range of 4 to 5 if your scale is from 1 to 5 and 8, 9, or 10 if your scale is from 1 to 10. Your CSAT score is the total number of positive responses divided by the total number of responses. So if you survey 100 people and 80 people give you a positive score that’s a CSAT score of 80%.
+
+### What’s a good CSAT score?
+
+A good CSAT score is typically between 75% to 85%. According to [Fullview](https://www.fullview.io/blog/csat-benchmarks-by-industry#toc-csat-benchmarks-for-software-companies), the average CSAT score for SaaS companies is 78%.
+
+## 6. Customer Effort Score (CES)
+
+### What is a CES?
+
+Another variation on the NPS and CSAT theme, your customer effort score, or CES, is a measure of how easy, or not, your product is to use. This is something of a leading indicator: a low CES score suggests you will probably have low retention because your product is hard to use – unless, miraculously, people are using it in spite of it being badly designed. As such, this is a really good metric to measure at a startup once you’ve found product-market fit.
+
+### How is it useful?
+
+This is a crucial metric for e-commerce and SaaS companies. If you’ve built a product or platform that should be easy to use and spark joy, then CES is a hugely important metric. On the flipside, if your business has very limited touchpoints with customers, if you run an insurance firm, for example, then measuring your CES might not be that revelatory.
```

Comment:
```suggestion
### What is a CES?

Your customer effort score, or CES, is a measure of how easy, or not, your product is to use. This is a crucial metric if "ease of use", rather than features, is the main differentiator between your product and your competition.

### Why is it useful?

This is something of a leading indicator: a low CES score suggests you will probably have low retention because your product is hard to use – unless, miraculously, people are using it in spite of it being badly designed. As such, this is a really good metric to measure at a startup once you’ve found product-market fit.
```